### PR TITLE
feat: new nav styles refinement

### PR DESCRIPTION
**Pull request by ackernaut**
- Refines alignment of nav items
- Refines border, text styles etc.
- Removes ‘Organizations’ nav item from mobile
- Updates the icon for Sign Out on mobile

### CODE REVIEW

File: packages/client/components/DashNavList/DashNavList.tsx [hunk 1]
```diff
@@ -60,32 +60,33 @@ const DashNavList = (props: Props) => {
...
+            className='mb-3 h-full w-full rounded-lg border border-solid border-slate-400'
           >
-            <div className='flex flex-wrap items-center'>
-              <div className='flex min-w-0 flex-1 flex-wrap items-center justify-between'>
-                <span className='text-md pl-2 font-medium leading-6 text-slate-700 '>
-                  {org.name}
-                </span>
-                <div className='flex w-auto justify-end px-0 text-right'>
-                  <Tag tier={org.tier}>{upperFirst(org.tier)}</Tag>
+            <div className='border-b border-solid border-slate-300 p-2'>
```
Comment (ackernaut) [inline packages/client/components/DashNavList/DashNavList.tsx:70]: Had to wrap here to have an inner border that went all the way across, but the inner components are the same

File: packages/client/components/Dashboard/MobileDashSidebar.tsx [hunk 2]
```diff
@@ -205,52 +127,114 @@ const MobileDashSidebar = (props: Props) => {
               />
               <NavItem
                 onClick={handleMenuClick}
-                icon={'forum'}
+                icon={'exit_to_app'}
```
Comment (ackernaut) [inline packages/client/components/Dashboard/MobileDashSidebar.tsx:130]: Removed wrappers here, but inner components are the same

**Review by nickoferrall — APPROVED** [review]
This looks great! I'm happy to merge all of these PRs into master now if you're happy too?

**Comment by ackernaut** [comment]
What do you think about creating a final branch from latest master, then merging everything into that branch to test? (Or have you pulled in master again at any point?). I want to check that there isn’t anything weird with the side nav layout that I had to do to enable an optional global banner. Happy to help test!

**Comment by nickoferrall** [comment]
Sounds good! I'll merge everything into one branch, and then we can test everything before merging with master

**Pull request merged**